jupyter-pytorch-full/rockcraft.yaml

```diff
@@ -7,28 +11,31 @@ description: |
 
   Both PyTorch and Jupyter are installed in Conda environment, which is
   automatically activated.
-version: "1.8.0"
+version: "v1.10.0-rc.1"
 license: Apache-2.0
-base: ubuntu@20.04
-run-user: _daemon_
+base: ubuntu@22.04
+
 services:
   jupyter:
     override: replace
     summary: "jupyter-pytorch-full service"
     startup: enabled
+    command: ./jupyter lab --notebook-dir="/home/jovyan" --ip=0.0.0.0 --no-browser --port=8888 --ServerApp.token="" --ServerApp.password="" --ServerApp.allow_origin="*" --ServerApp.allow_remote_access=True --ServerApp.base_url="/" --ServerApp.authenticate_prometheus="False"
+    working-dir: /opt/conda/bin/
+    user: jovyan
     environment:
        NB_USER: jovyan
-       NB_UID: 1001
```

Comment:
Is there a reason why we removed this env variable? I see it is set in the [base Dockerfile](https://github.com/kubeflow/kubeflow/blob/v1.10.0-rc.1/components/example-notebook-servers/base/Dockerfile#L12C1-L12C16).